### DISCUSSION TITLE
Quick Question about track number

### DISCUSSION THREAD

**Discussion by AmericanVaultBoy** [TT-ReBORN/Georgia-ReBORN#55] [Q&A] [answered]
Hello, I'm new to modding with foobar but I think Georgia-ReBorn is almost perfect but it's doing something similar to what the original Georgia did which is displaying the track number or folder number or something. I don't like using the grouping and even if I did I still wouldn't know how to disable it. I've linked an image of some of the songs but they all look like that. I'm not sure how to edit it because it doesn't have the columns on the top like the normal foobar. Sorry if it's a noobie question but I'm just not sure how to change it.
![unknown](https://user-images.githubusercontent.com/7930055/154824367-153323e5-ee0c-4faf-9389-94428173eeb6.png)

**Comment by TT-ReBORN** [comment, marked as answer]
Hi @AmericanVaultBoy,

these are the track numbers. if you don't like them you can open Panel_Playlist.js ( foobar2000\profile\georgia-reborn\js\CaTRoX_QWR )
and as for the current version right now go to line [5180](https://github.com/TT-ReBORN/Georgia-ReBORN/blob/9d162528177ef16a61d088743403b046845d9f09/profile/georgia-reborn/js/CaTRoX_QWR/Panel_Playlist.js#L5180)

or search for this:
`var title_query = g_properties.show_header ? track_num_query + (pref.show_artist_playlistRow && pref.show_album_playlistRow ? '  %artist% - %album% - ' + ' %title%[ \'(\'%original artist%\' cover)\']' : pref.show_artist_playlistRow ? '  %artist% - ' + ' %title%[ \'(\'%original artist%\' cover)\']' : pref.show_album_playlistRow ? '  %album% - ' + ' %title%[ \'(\'%original artist%\' cover)\']' : '  %title%[ \'(\'%original artist%\' cover)\']') : '     %artist% - %album% - ' + track_num_query + ' %title%[ \'(\'%original artist%\' cover)\']';
`

and replace it with this:
`var title_query = g_properties.show_header ? /* track_num_query */ '   ' + (pref.show_artist_playlistRow && pref.show_album_playlistRow ? '  %artist% - %album% - ' + ' %title%[ \'(\'%original artist%\' cover)\']' : pref.show_artist_playlistRow ? '  %artist% - ' + ' %title%[ \'(\'%original artist%\' cover)\']' : pref.show_album_playlistRow ? '  %album% - ' + ' %title%[ \'(\'%original artist%\' cover)\']' : '  %title%[ \'(\'%original artist%\' cover)\']') : '     %artist% - %album% - ' + track_num_query + ' %title%[ \'(\'%original artist%\' cover)\']';`

Save and reload foobar.

-TT

**Comment by AmericanVaultBoy** [comment]
@TT-ReBORN 

Ok, it worked but I also had to remove that "+ track_num_query" it's like the 3rd to the last thing in the thing you said to copy. Thank you so much. now it's showing the play button into the band title. 
![image](https://user-images.githubusercontent.com/7930055/154863372-076f52c4-4b5e-4921-bcda-9a64b2ed4215.png)
It's not a big deal though I could live with it. I tried messing with the code to fix it myself but I am not much of a programmer. 

Also, just a general question if I could get a little bit more of your time why isn't Georgia Reborn like a normal window? In normal foobar when I have it full screen I can just click and hold the top of the window and move it to make it smaller and to move it. In this, I have to first double click it and then it changes it to a smaller window. Is there any way to change that back too? Thank you so much for the help though.

**Reply by TT-ReBORN** [reply]
Ok, so you're not using the group header, than just replace the second last "track_num_query" with:  ` ' ' ` 
You can add or remove whitespace/s till the play icon has enough padding...

Double click on the top menu = Maximize ( fills automatically your display, but your taskbar is still visible ), 
double clicking again will go to your last player size. 
Fullscreen is the 🗖 button in the top menu on the right corner side. 
This will completely fill your display like the Youtube/Vimeo fullscreen function. 

You can use hold mouse wheel to drag the player around or you can hold left click on top menu.
To manually resize the player you can use the bottom left or right corner and resize it. While you're in maximize or fullscreen mode it's obviously not possible to resize it, because these two modes automatically resize/fill, otherwise it would beat the purpose behind these functions.

The best way to resize it is to use the proportional predefined player sizes in top menu Options > Player size.

-TT

**Reply by AmericanVaultBoy** [reply]
> Ok, so you're not using the group header, than just replace the second last "track_num_query" with: `' '` You can add or remove whitespace/s till the play icon has enough padding...
> 
> Double click on the top menu = Maximize ( fills automatically your display, but your taskbar is still visible ), double clicking again will go to your last player size. Fullscreen is the 🗖 button in the top menu on the right corner side. This will completely fill your display like the Youtube/Vimeo fullscreen function.
> 
> You can use hold mouse wheel to drag the player around or you can hold left click on top menu. To manually resize the player you can use the bottom left or right corner and resize it. While you're in maximize or fullscreen mode it's obviously not possible to resize it, because these two modes automatically resize/fill, otherwise it would beat the purpose behind these functions.
> 
> The best way to resize it is to use the proportional predefined player sizes in top menu Options > Player size.
> 
> -TT

Adding spaces worked perfectly. THANK YOU. For the second part I mostly use it to move the full screen to another monitor, I could send you a video in normal foobar to show you what I mean but it sounds like you changed it purposefully which makes sense. I'm sure most people don't move the window. Which I can live with. 

Sorry to ask more questions but these should be it if you don't mind. 
1. Is there any reason that scrolling through a playlist isn't snappy and very slow. It feels laggy and sometimes when I download lyrics for an entire playlist it just crashes but it might be just me. Am I doing something wrong or is it just how it is?

2. Is there a way to edit the lyrics panel? I think my settings and the way I color it is slightly different than I'd like and I've clicked everywhere. 


Sorry again for bothering you I think what you guys are doing is awesome. The whole theme here is like wayyy greater than the sum of its parts. When I installed the lyrics thing it couldn't find basically anything. Play count is shown in more places. Everything I've used is better. If there's any way I can support tell me.

**Reply by TT-ReBORN** [reply]
Actually I'm working on the upcoming update which has mostly a complete code rewrite which will greatly benefit in performance and lots of new stuff, it will be a new experience for sure...

For playlist scrolling you should check top menu Options > Player controls > Scrollbar settings > Playlist > Mouse wheel scroll steps and smooth duration. You can play around with these two settings. The lag depends on resolution/player size, CPU and I guess is limited by the performance of GDI+. Maybe performance will be much better in the upcoming update, you be the judge...

Yea the lyric component is a bit unstable. You should disable/remove the NEW DB: Deezer source from the search order in 
File > Preferences ( Ctrl + P ) > Tools > Lyric Show 3. NetEase and QQ Music ( Chinese servers ) should be already removed. 
The crashes were also reported by some users, this will be default in the upcoming update.

Lyric colors can be modified in the Panel properties ( if top menu Options > Settings > Disable right click is unticked, right click on top menu or lower bar > Panel properties  ) under these:
Georgia-ReBORN - Lyrics: Text Color
Georgia-ReBORN - Lyrics: Text Highlight Color

I will announce the new update when it's ready in the Georgia-ReBORN thread at HydrogenAudio:
https://hydrogenaud.io/index.php?topic=121047.0
and here in the general discussion thread, stay tuned.

-TT
